documentation/AI_Events.txt

```diff
@@ -31,6 +31,7 @@ WEATHER_CHANGE - Entity, weather, weather element
 EXPERIENCE_POINTS - Entity, exp
 
 TICK - Entity
-EFFECTS_TICK - Entity
+COMBAT_TICK - Mob
```

Comment:
ahh don't delete EFFECTS_TICK, as I put it back in yesterday

Reply:
done